h/static/scripts/vendor/annotator.auth.js

```diff
@@ -1,13 +1,13 @@
 
 /*
-** Annotator v1.2.9-dev-b091a74
+** Annotator v1.2.9-dev-368b5cf
```

Comment:
Haven't I argued in favor of these empty updates, so that it's clear from looking at any one file which version of annotator it all came from? Since the plugins aren't separately versioned they should all come from the same annotator version.